### ISSUE TITLE
Handling of homonyms? 

### ISSUE THREAD

**Issue by turley85** [AtlasOfLivingAustralia/alerts#253]
What is the best practice for managing homonyms?

Two commons homonyms in our alert lists are Prosopis and Salvinia - which are homonyms of both plants and animals. 

Should I edit supplied name to match the ALA taxonomy (e.g Prosopis L. and Salvinia Ség.)? Or should I leave it as provided (Prosopis and Salvinia)? And how will the alert system know we're searching of the plant (not animal) if left unmatched?

**Comment by turley85** [comment]
Eg. Salvinia is a synonym of this beetle https://bie.ala.org.au/species/https://biodiversity.org.au/afd/taxa/cd4ad7da-4869-4897-966d-6cbcdc129389#names 

This is the weeds we're seeking: https://bie.ala.org.au/species/https://id.biodiversity.org.au/node/apni/2908448

**Comment by turley85** [comment]
Eg. Prosopis animal: https://bie.ala.org.au/species/https://biodiversity.org.au/afd/taxa/d326831d-bf60-4146-8d07-e72391307b13#classification

Prosopis weed: https://bie.ala.org.au/species/https://id.biodiversity.org.au/taxon/apni/51436746

**Comment by turley85** [comment]
Related issue - here an unmatched taxa Phytoplama (kingdom Bacteria) is matching to the plant genus phytoplama 
<img width="641" alt="image" src="https://github.com/user-attachments/assets/287c2b06-4aaa-40b5-97e1-f17bfd5c4085">
[BioSecurity alert for xTestListMissingTaxa](https://lists.ala.org.au/speciesListItem/list/dr27453)

**Comment by kylie-m** [comment]
Hi @turley85 I've been chatting with the team on this, to my understanding it should be functioning as per the current R alerts - happy to chat about it more if needed, though it does sound like this would be an enhancement to the current setup

**Comment by turley85** [comment]
@kylie-m Yeah, this should be an enhancement. Make me wondering if R alerts for homonyms is just failing.

**Comment by kylie-m** [comment]
I have added the following note to our [Biosecurity Alerts Workflow](https://confluence.csiro.au/spaces/ALASD/pages/1913483552/ALA+Biosecurity+Alerts+Workflow) document

> To improve name matching in the Lists tool, it is strongly recommended to include a kingdom and class column.
